org/wildfly/undertow/servlet-container/capability.adoc

```diff
@@ -0,0 +1,24 @@
+NAME: org.wildfly.undertow.servlet-container
+
+DESCRIPTION: Undertow's servlet container service
+
+REGISTERED BY:
+
+  NAME: WildFly
+  URL:  https://github.com/wildfly/wildfly
+
+DYNAMIC: true
+
+PRIVATE: false
```

Comment:
Same comment here as for the UndertowService, Host and Server capabilities.

This one seems to have less API in it that is inappropriate, other than exposing Service itself and perhaps some injectors.